pylint/lint.py

```diff
@@ -922,8 +929,9 @@ def _do_check(self, files_or_modules):
     def expand_files(self, modules):
         """get modules and errors from a list of modules and handle errors
         """
+        print("running pylint")
         result, errors = utils.expand_modules(modules, self.config.black_list,
-                                              self.config.black_list_re)
+                                              self.config.black_list_re, self.lint_all)
```

Comment:
This is `self.config.lint_all`, not `self.lint_all`.

Reply:
Will I have to define the `lint_all` variable in the config.py file ?